src/main/java/io/github/ImpactDevelopment/installer/gui/AppWindow.java

```diff
@@ -50,7 +50,7 @@ public AppWindow(InstallationConfig config) {
         setContentPane(wrapper);
         setContent(loadingScreen());
         setSize(690, 420);
-        setResizable(false);
+        setResizable(true);
```

Comment:
:(

Reply:
oops that was for debugging - the widgets weren't fitting on the frame at one point

Reply:
Fixed